### ISSUE TITLE
webpagetest Test Error: Error configuring traffic-shaping on ubuntu 18.04

### ISSUE THREAD

**Issue by nutmix** [WPO-Foundation/wptagent#591]
Ive just installed the agent on a VPSCity ubuntu 18.04. I had to overcome several undocumented pre-requisits, such as specific sources.list.

Now I can see the agent in on the server, but when I try to run the first test I get:

webpagetest Test Error: Error configuring traffic-shaping

I saw some posts with this error on docker (im not using docker), and they suggested trying the following:

$ sudo modprobe ifb numifbs=1

Im not sure if this is relevant in don docker cases, however, this gives:

modprobe: FATAL: Module ifb not found in directory /lib/modules/4.15.0

Any ideas how to get the agent working?

**Comment by Gujjala** [comment]
yeah, there are quite lot of undocumented dependencies while starting webpagetest agent. I'm getting above error.  getting error 04:21:49.069 - Error configuring traffic shaping, make sure it is installed.
